### PR TITLE
Added nautical scale

### PR DESCRIPTION
**Pull request by christiangrs**
Added nautical scale with nautical miles and yards.

### CODE REVIEW

File: src/control/Control.Scale.js [hunk 1]
```diff
@@ -62,6 +66,9 @@ export var Scale = Control.extend({
 		if (options.imperial) {
 			this._iScale = DomUtil.create('div', className, container);
 		}
+		if (options.nautical) {
+			this._iScale = DomUtil.create('div', className, container);
```
Comment (Falke-Design) [inline src/control/Control.Scale.js:70]: Change `_iScale` to `_nScale`

File: src/control/Control.Scale.js [hunk 2]
```diff
@@ -106,6 +116,21 @@ export var Scale = Control.extend({
...
+		    maxMiles, miles, yards;
+
+		if (maxYards > 2025) {
+			maxMiles = maxYards / 2025.37;
+			miles = this._getRoundNum(maxMiles);
+			this._updateScale(this._iScale, miles + ' nmi', miles / maxMiles);
+
+		} else {
+			yards = this._getRoundNum(maxYards);
+			this._updateScale(this._iScale, yards + ' yd', yards / maxYards);
```
Comment (Falke-Design) [inline src/control/Control.Scale.js:130]: Change `_iScale` to `_nScale`

File: src/control/Control.Scale.js [hunk 3]
```diff
@@ -106,6 +116,21 @@ export var Scale = Control.extend({
...
 	},
 
+	_updateNautical: function (maxMeters) {
+		var maxYards = maxMeters * 1.0936133,
+		    maxMiles, miles, yards;
+
+		if (maxYards > 2025) {
+			maxMiles = maxYards / 2025.37;
+			miles = this._getRoundNum(maxMiles);
+			this._updateScale(this._iScale, miles + ' nmi', miles / maxMiles);
```
Comment (Falke-Design) [inline src/control/Control.Scale.js:126]: Change `_iScale` to `_nScale`

**Review by Falke-Design — CHANGES_REQUESTED** [review]
Please revert the changes in the `reference.html` it is generated automatically.

Would look like that:
image

Running sample: https://plnkr.co/edit/ODFqo4Gs6coMfovr

**Review by Falke-Design — APPROVED** [review]
Checked with google converter meter to yard, yard to nauticalmiles. It uses the same faktors for converting.

**Comment by jonkoops** [comment]
Is this a functionality we'd want to include in the core? This seems like it could quite easily be a plugin. @mourner WDYT?

**Review by christiangrs — COMMENTED** [review]
Reviewed

**Comment by Falke-Design** [comment]
> Is this a functionality we'd want to include in the core? This seems like it could quite easily be a plugin. @mourner WDYT?

@mourner? Closing or merging?

**Comment by jonkoops** [comment]
I'm closing this PR as I don't believe this functionality needs to be in the core of Leaflet. Feel free to create a plugin that allows users to use this functionality if it is useful to them.

**Pull request closed without merging**